src/main/java/cinema/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package cinema.service.impl;
+
+import cinema.dao.ShoppingCartDao;
+import cinema.dao.TicketDao;
+import cinema.injections.Inject;
+import cinema.injections.Service;
+import cinema.model.MovieSession;
+import cinema.model.ShoppingCart;
+import cinema.model.Ticket;
+import cinema.model.User;
+import cinema.service.ShoppingCartService;
+
+@Service
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+    @Inject
+    private ShoppingCartDao shoppingCartDao;
+    @Inject
+    private TicketDao ticketDao;
+
+    @Override
+    public void addSession(MovieSession movieSession, User user) {
+        Ticket ticket = new Ticket(movieSession, user);
+        ticketDao.add(ticket);
```

Comment:
if there is no shopping cart for the user - we still save ticket to the db, is it ok?